README.md

```diff
@@ -6,29 +6,41 @@
 
 Features:
 
```

Comment:
Not sure if there was a need to change these as `-` and `*` are both valid Markdown for lists.

Reply:
I agree. This is just an artifact of my editor's consistency setting. I can revert these changes if you'd prefer.